pkg/tests/api/correlations/correlations_update_test.go

```diff
@@ -145,7 +154,12 @@ func TestIntegrationUpdateCorrelation(t *testing.T) {
 		res := ctx.Patch(PatchParams{
 			url:  fmt.Sprintf("/api/datasources/uid/%s/correlations/%s", readOnlyDS, "nonexistent-correlation-uid"),
 			user: adminUser,
-			body: `{}`,
+			body: `{
+				"config": {
+					"type": "query",
+					"field": "fieldName",
+					"target": { "expr": "bar" }
+			}}`,
```

Comment:
same here, this shouldn't be needed
```suggestion
			body: `{}`,
```

Reply:
True. I removed it.